playbook/app/pb_kits/playbook/pb_link/link.html.erb

```diff
@@ -0,0 +1,21 @@
+<% link_content = proc do %>
+  <% if object.icon.present? %>
+    <%= pb_rails("icon", props: { icon: object.icon, fixed_width: true, size: "xs", margin_right: "xxs" }) %>
+  <% end %>
+  <%= object.content %>
+  <% if object.icon_right.present? %>
+    <%= pb_rails("icon", props: { icon: object.icon_right, fixed_width: true, size: "xs", margin_left: "xxs" }) %>
+  <% end %>
+<% end %>
+
+<% if object.tag == "a" %>
+  <%= pb_content_tag(object.tag, { href: object.href }) do %>
+    <%= link_content.call %>
+  <% end %>
+<% else %>
+  <%= pb_content_tag(:a, { href: object.href }) do %>
+    <%= content_tag(object.tag) do %>
+      <%= link_content.call %>
+    <% end %>
+  <% end %>
+<% end %>
```

Comment:
For this piece, wouldn't having a default in the .rb file suffice to render the 'a' tag if none is explicitly passed in? I feel like this conditional in the html.erb shouldn't be needed

Reply:
@nidaqg, I made it so an <a> tag wouldn't be wrapped around an <a> tag, but it would wrap around any other tags. 

I could make it so that the tag's default is a <span>, though. It would certainly make this code look a lot cleaner! Let me know if you think it'd make sense to go this direction!

Reply:
Let's go ahead and leave this logic for now. I think it makes sense!